CRM/Member/Page/UserDashboard.php

```diff
@@ -76,11 +76,21 @@ public function listMemberships() {
 
   /**
    * Helper function to build appropriate Member links
+   *
+   * @array &members
+   * bool isActiveMembers
    */
-  public function buildMemberLinks(&$members) {
+  public function buildMemberLinks(&$members, $isActiveMembers) {
```

Comment:
@shaneonabike this is causing a test failure:
```
ArgumentCountError: Too few arguments to function CRM_Member_Page_UserDashboard::buildMemberLinks(), 1 passed in /home/homer/buildkit/build/build-2/web/sites/all/modules/civicrm/CRM/Member/Page/UserDashboard.php on line 69 and exactly 2 expected
```

You could fix this by making the param optional:
```suggestion
  public function buildMemberLinks(&$members, $isActiveMembers = FALSE) {
```

Reply:
Awesome suggestion thanks for your help and sorry for the late update!